types/src/jwks/rsa/insecure_test_jwk.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
insecure in which way?

Reply:
the private key is in the repo :P.  this insecure one is for our smoke tests so that we can make jwts in the tests.

the secure one is for load testing of pepper and prover.